### PR TITLE
Enable FindBugs for the plugin

### PR DESCRIPTION
**Pull request by oleg-nenashev**
@reviewbybees

### CODE REVIEW

**Comment by oleg-nenashev** [comment]
Te code formatting in the plugin is horrible BTW.

File: src/main/java/hudson/security/AuthorizationMatrixProperty.java [hunk 1]
```diff
@@ -300,9 +306,9 @@ public Object unmarshal(HierarchicalStreamReader reader,
 				reader.moveDown();
 				reader.getValue(); // we used to use this but not any more.
 				reader.moveUp();
-				prop = reader.peekNextChild();
+				prop = reader.peekNextChild(); // We check the next field
 			}
-			else if ("blocksInheritance".equals(prop)) {
```
Comment (stephenc) [inline src/main/java/hudson/security/AuthorizationMatrixProperty.java]: Seems like a non-findbugs related change... not saying it is incorrect... rather that it seems like one

Comment (oleg-nenashev) [inline src/main/java/hudson/security/AuthorizationMatrixProperty.java]: It's a FindBugs one. Otherwise FindBugs complains that the assigned value from the previous if clause is never used. Probably it should be registered as a separate functional bug, but I was not able to find related issues

Comment (stephenc) [inline src/main/java/hudson/security/AuthorizationMatrixProperty.java]: Sounds like something that should be merged but in a separate pull request then

Comment (oleg-nenashev) [inline src/main/java/hudson/security/AuthorizationMatrixProperty.java]: Agreed

**Comment by oleg-nenashev** [comment]
@stephenc 
Created https://issues.jenkins-ci.org/browse/JENKINS-29527 and removed the change from this PR

**Comment by stephenc** [comment]
:bee:

**Comment by oleg-nenashev** [comment]
@reviewbybees done
